content/en/docs/components/katib/experiment.md

```diff
@@ -379,18 +401,19 @@ Katib supports the following algorithm settings:
 </div>
 
 <a id="enas"></a>
+
 #### Neural Architecture Search based on ENAS
 
 {{% alert title="Alpha version" color="warning" %}}
-Neural architecture search is currently in <b>alpha</b> with limited support. 
-The Kubeflow team is interested in any feedback you may have, in particular with 
+Neural architecture search is currently in <b>alpha</b> with limited support.
+The Kubeflow team is interested in any feedback you may have, in particular with
 regards to usability of the feature. You can log issues and comments in
 the [Katib issue tracker](https://github.com/kubeflow/katib/issues).
 {{% /alert %}}
 
 The algorithm name in Katib is `enas`.
```

Comment:
Maybe move this sentence up so it appears after the section headline (subheading), like in other algorithm descriptions?

```suggestion
The algorithm name in Katib is `enas`.
```